wiki/History_of_osu!/2014/id.md

```diff
@@ -1,29 +1,44 @@
+
+![](img/2014-03_01.jpg "Latar belakang bawaan untuk beatmap tanpa latar belakang yang spesifik*")
+
+## April
 
-![](img/2014-03_01.jpg "playfield.jpg sudah tidak ada lagi")
+"osu! Monthly Fanart Contest" pertama kalinya diumumkan pada tanggal 3 April 2014. Kontes tersebut merupakan kontes informal yang diselenggarakan oleh [LuigiHann](https://osu.ppy.sh/users/1079) yang akan menjalankan kontes tersebut setiap bulan dan memberikan dua orang pemenang penghargaan berupa [osu!supporter](https://osu.ppy.sh/home/support) (satu pemenang dipilih oleh publik dan satu pemenang lagi dipilih oleh staf osu!) \[7\]. Hasil dari edisi pertama kontes disimpulkan dalam sebuah [*newspost*](https://osu.ppy.sh/home/news/2014-05-21-first-official-osu-fanart-contest-results) pada tanggal 21 Mei 2014, dimana *player* [kunny](https://osu.ppy.sh/users/3931585) dan [DumplingYumYum](https://osu.ppy.sh/users/1715930) masing-masing memenangkan juara satu dan dua.
 
-## Juli
+## Juni
 
-peppy menghapus batasan unduhan\*!
+Pada tanggal 18 Juni 2014, [peppy](https://osu.ppy.sh/users/2) mengumumkan bahwa batasan pengunduhan *beatmap* akan dihapus untuk semua *player*; namun, akan tetap ada "batasan kewarasan (sanity check)" yang akan mencegah *bot* dari pengunduhan beatmap dalam jumlah yang berlebihan \[9\].
 
-Tautan Terkait:
+## Agustus
 
-- [Download restrictions removed](https://osu.ppy.sh/home/news/2014-06-18-download-restrictions-removed)
+Pada tanggal 21 Agustus 2014, sebuah [*newspost*](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) mengumumkan bahwa akan diadakannya restrukturisasi [Beatmap Appreciation Team (BAT)](/wiki/Modding/Beatmap_Appreciation_Team), dimana dirincikan bahwa telah dilakukannya pemecahan tim asli BAT menjadi dua tim, yaitu Beatmap Appreciation Team dan [Quality Assurance Team (QAT)](/wiki/Modding/Quality_Assurance_Team) \[10\].
 
 ## Oktober
 
-Seperti biasa, banyak hal baru ditambahkan ke osu!. Salah satunya adalah layar ketika masuk sudah diperbarui. Hal lain yang baru adalah sistem *scrolling* dalam [osu!mania](/wiki/Game_mode/osu!mania). Sistem baru ini berdasarkan pada bpm beatmap daripada *scroll* yang sudah ditetapkan! Intro dan outro baru ketika memasuki game, daripada langsung memainkan musik beatmap saat memasuki game, tema lagu yang luar biasa telah ditambahkan ke osu!. Dan kebanyakan orang mengucapkan osu! sebagai \\oss!\\, bukan \\osU!\\.
+Seperti biasa, banyak hal baru ditambahkan ke osu!. Salah satunya adalah layar ketika masuk sudah diperbarui. Hal lain yang baru adalah sistem *scrolling* dalam [osu!mania](/wiki/Game_mode/osu!mania). Sistem baru ini berdasarkan pada BPM beatmap daripada *scroll* yang sudah ditetapkan! Intro dan outro baru ketika memasuki game, daripada langsung memainkan musik beatmap saat memasuki game, tema lagu yang luar biasa telah ditambahkan ke osu!. Dan kebanyakan orang mengucapkan osu! sebagai \\oss!\\, bukan \\osU!\\.
```

Comment:
```suggestion
Berbagai fitur baru ditambahkan ke osu! pada bulan ini, mulai dari layar masuk (*sign-in screen*) baru, sistem *scrolling* [osu!mania](/wiki/Game_mode/osu!mania) baru yang berdasar pada BPM, animasi intro dan outro baru, hingga lagu tema baru yang akan terdengar setiap kali osu! dinyalakan.
```

Reply:
(i checked the original english article and it appears that the actual paragraph here has been drastically simplified in which a lot of things have been removed)